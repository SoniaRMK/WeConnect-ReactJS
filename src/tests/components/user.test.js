import {shallow, mount} from 'enzyme';
import React from 'react';
import User from '../../components/user/user';

describe('user component',() => {
    let wrapper;

    it('should match snapshot',() =>{
        wrapper = shallow(<User/>)
        expect(wrapper).toMatchSnapshot();
    })

})
