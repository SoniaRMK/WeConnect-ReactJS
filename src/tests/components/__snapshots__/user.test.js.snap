// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`user component should match snapshot 1`] = `
<Route
  render={[Function]}
/>
`;
